fmriprep/workflows/bold/outputs.py

```diff
@@ -220,6 +220,7 @@ def init_func_derivatives_wf(
                 't2star_std',
                 'bold2anat_xfm',
                 'anat2bold_xfm',
+                'hmc_xforms',
```

Comment:
And this one to the docstring inputs

Reply:
This function doesn't have docstrings for input/output nodes.